frontend/components/buttons/RevealButton/RevealButton.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import Button from "components/buttons/Button";
+import TooltipWrapper from "components/TooltipWrapper";
+
+export interface IRevealButtonProps {
+  showBoolean: boolean;
```

Comment:
Minor name change: it's better practice to name a boolean in the form of a question. Let's do something like `shouldShow` or `isShowing`.